__tests__/app.test.js

```diff
@@ -239,6 +239,46 @@ describe('/api/articles/:article_id', () => {
 	});
 });
 
+describe('/api/articles/:article_id/comments', () => {
+	describe('GET', () => {
+		test('200: responds with an array of commets for the given article_id', () => {
+			return request(app)
+				.get('/api/articles/3/comments')
+				.expect(200)
+				.then((response) => {
+					const allComments = response.body.comments;
+					expect(Array.isArray(allComments)).toBe(true);
+					expect(allComments.length > 0).toBe(true);
+					allComments.forEach((comment) => {
+						expect(comment).toHaveProperty('body', expect.any(String));
+						expect(comment).toHaveProperty('votes', expect.any(Number));
+						expect(comment).toHaveProperty('author', expect.any(String));
+						expect(comment).toHaveProperty('comment_id', expect.any(Number));
+						expect(comment).toHaveProperty('created_at');
+					});
```

Comment:
We do need another test here. What if an article exists but doesn't have any comments? We want this to be a 200 and send the empty array